### DISCUSSION TITLE
Mega board in pack.

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#204] [Q&A] [answered]
How much longer do you guys see supporting the mega board diy kits in the pack itself? Or is it always going to utilize that chip style I/o's until you guys revise into a new board and chip? 

I'm updating my wand with the new board chip and removing the nano to utilize the e-proms. 

Do you recommend doing the same on the pack side for more long term support?

**Comment by gpstar81** [comment, marked as answer]
The diy pack will always be supported because the GPStar pack board is utilising the same ATmega2560 chipset. It was only the ATmega328p in the diy wands which was the only issue.
